app/models/git/exercise.rb

```diff
@@ -244,6 +244,12 @@ def dir = "exercises/#{exercise_type}/#{exercise_slug}"
     memoize
     def approaches = Git::Exercise::Approaches.new(exercise_slug, exercise_type, git_sha, repo:)
 
+    # TODO: - Drive this from a representer_version key in a
+    # track's config.json exercise block.
```

Comment:
Shouldn't this be the exercise's `.meta/config.json` file? I'd prefer to keep the track's `config.json` as clean as possible.